.github/workflows/continuous-deployment-server.yml

```diff
@@ -0,0 +1,105 @@
+name: Continuous Deployment indy-vdr-proxy-server
+
+on:
+  push:
+    branches: [main]
+
+  workflow_dispatch:
+
+env:
+  DH_USERNAME: ${{ secrets.DOCKER_HUB_LOGIN }}
+  DH_TOKEN: ${{ secrets.DOCKER_HUB_PWD }}
+  IMAGE_NAME: "indy-vdr-proxy-server"
+
+jobs:
+  package-finder:
+    runs-on: ubuntu-latest
+    outputs:
+      packages: ${{ steps.set-packages.outputs.packages }}
+    steps:
+      - name: Find packages
+        id: set-packages
+        run: |
+          echo "::set-output name=packages::$(jq -nc '[ "server" ]')"
+
+  release-please:
```

Comment:
Don't use release-please here: it will have conflicts with the job defined in continuous-integration.yml.

The proper way to handle this would be to make image generation part of the release-please job defined in continuous-integration.yml. I just wanted to keep this PR simple to move forward faster, and that is why I suggested to simply publish an image with `dev`tag in order to not deal with the release-please process.

If you really want to do this management in this PR, then you'll need to:
1. Add everything you have here from line 66 into the end of current continuous-integration.yml
2. You'll need to add also to the `release-please` job the `release_json` line you've created here
3. All these steps must be run only in case of a release related to `server`. This is very important to note because the release-please job will be executed every time any of the `client` or the `server` are about to be released. Therefore, you need to add a condition in order for these steps to run only if matrix.package = 'server' (I don't know the syntax for multiple conditions; we'll need to investigate that)
4. Delete this file, since we'll only work with the continuous-integration.yml one

I hope I didn't forget any step

Reply:
Hi! @genaris , i keep the deployment indy-vdr-server simple,  i prefer a test this idea working good , after i modify the continuos-integration file with you consideration.